server/tests/tasksTest.js

```diff
@@ -0,0 +1,31 @@
+import regeneratorRuntime from 'regenerator-runtime';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker'; // Fake data generator
+import app from '../app';
+// import dotenv from 'dotenv';
+
+// dotenv.config();
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('Tasks: POST', () => {
+  it('should create a new Task in storage', (done) => {
+    // Sample Task
+    const newTask = {
+      title: faker.lorem.words(3),
+      description: 'Testing: new task created in the database...' + faker.lorem.sentence(7),
+    };
+
+    chai
+      .request(app)
+      .post('/api/v1/tasks')
+      .send(newTask)
+      .end((err, res) => {
+        expect(res.statusCode).to.equal(201);
+        done();
+      });
+  });
+
```

Comment:
#### Please finish the tests.

![image](https://user-images.githubusercontent.com/52471122/76370936-33b8e200-6341-11ea-9e50-cca6ea50534a.png)